packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -133,12 +133,13 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   const defaultNumberOfChatMessagesToReload = 5;
   /* @conditional-compile-remove(file-sharing) */
   const [downloadErrorMessage, setDownloadErrorMessage] = React.useState('');
-  /* @conditional-compile-remove(image-gallery) */
+  /* @conditional-compile-remove(image-overlay) */
   const [fullSizeAttachments, setFullSizeAttachments] = useState<Record<string, string>>({});
-  /* @conditional-compile-remove(image-gallery) */
-  const [galleryImages, setGalleryImages] = useState<Array<ImageGalleryImageProps>>([]);
-  /* @conditional-compile-remove(image-gallery) */
-  const [isImageGalleryOpen, setIsImageGalleryOpen] = useState<boolean>(false);
+  /* @conditional-compile-remove(image-overlay) */
+  const [overlayImageItem, setOverlayImageItem] =
+    useState<{ imageSrc: string; title: string; titleIcon: JSX.Element; downloadFilename: string }>();
```

Comment:
`{ imageSrc: string; title: string; titleIcon: JSX.Element; downloadFilename: string }` should we typealias this?